src/components/NcActions/NcActions.vue

```diff
@@ -1130,7 +1131,7 @@ export default {
 		 */
 		boundariesElement: {
 			type: Element,
-			default: () => document.querySelector('body'),
+			default: () => document.querySelector('#app-content-vue') ?? document.querySelector('body'),
```

Comment:
Whoops, we didn't check, and that's breaking NcActions in the NcAppNavigation and NcAppSidebar.Shouldn't it be `#content-vue`, as in NcContent?

a | b
-- | --
![image](https://github.com/user-attachments/assets/39eb7a0a-5709-4773-94b2-d7af4beb71ac)![image](https://github.com/user-attachments/assets/097c210c-d3c2-429a-902e-b3eeaf675047)